metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alert.engine.eventcondition.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
+}
+
+message MetricSelection {
+  Attribute metric_attribute = 1;
+  MetricAggregationFunction metric_aggregation_function = 3;
+  string metric_aggregation_interval = 4; //iso duration string PT15S
+  Filter filter = 5;
+}
+
+message Attribute {
+  string key = 1;
```

Comment:
nit: can we rename this to attribute_key and attribute_scope?

Reply:
As we are using the same name in our attribute definition, it will be easy to explain (https://github.com/hypertrace/config-bootstrapper/blob/main/config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_service_attributes.conf#L13), lets go with the same. What do you think?

Reply:
sure